manifests/init.pp

```diff
@@ -18,6 +18,13 @@
 		extract_path => "/usr/local/bin",
 		creates      => "/usr/local/bin/sera",
 	}
+	
+	file { "/usr/local/bin/sera": 
+		owner => "root",
+		group => "root",
+		mode => 0755;
```

Comment:
I guess the `;` isn't needed, I haven't seen it done like that for other resources.